### DISCUSSION TITLE
Installation instruction

### DISCUSSION THREAD

**Discussion by sempasha** [MvBonin/wingpanel-community-indicators#4] [Q&A]
[Installation instruction](https://github.com/MvBonin/switchboard-plug-indicators/blob/fac9d48762ced77ae3750ddb861ca5ed2e793fe3/README.md#easy-installation) refers to .deb. files.

>  Download .deb file and install it via
> ```sudo apt install ./FILENAME.deb```

Unfortunately I haven't found any .deb file in the project.
Do I need to build .deb file myself?

**Comment by mufidu** [comment]
There is a .deb file in the repo.

**Reply by sempasha** [reply]
Now I see, thank you for opening my eyes.
